test/git.test.js

```diff
@@ -83,6 +83,18 @@ describe('git', function () {
       });
     });
 
+    it('skips any excluded commit types', function () {
+      Sinon.stub(CP, 'execAsync')
+        .onFirstCall().returns(Bluebird.resolve('v1.2.3'))
+        .onSecondCall().returns(Bluebird.resolve(VALID_COMMITS));
+
+      return Git.getCommits({ exclude: ['chore', 'style']})
+        .then(function (commits) {
```

Comment:
Can you align the `.then` with the `return` like how the other tests have it? Here and in `package.test.js`?